TipViewController.swift

```diff
@@ -45,9 +45,11 @@ class TipViewController: UIViewController {
 
         tipTitleLabel.text = tip.title
         tipDescriptionLabel.text = tip.description
+        tipDescriptionLabel.textAlignment = .center
```

Comment:
Minor: Configuration can be moved at label creation

Reply:
Ok, I will add this change. Thank you!